src/MatrixFields/field_matrix_iterative_solver.jl

```diff
@@ -375,19 +379,31 @@ end
 function StationaryIterativeSolve(;
     P_alg = nothing,
     n_iters = 1,
+    debug = nothing,
     correlated_solves = false,
     eigsolve_kwargs = (;),
 )
+    B = any(("error_norm", "spectral_radius")) do x
+        occursin(x, get(ENV, "JULIA_DEBUG", ""))
+    end
```

Comment:
```suggestion
    B = Logging.min_enabled_level(Logging.current_logger()) <= Logging.Debug
```
If you want to avoid needing to modify `ENV` for the unit test, you can do something like this. There's also the more specific `Logging.shouldlog` function that can check whether to log the `error_norm` and `spectral_radius` groups, but that's already called internally by the logger, and there's no benefit to doing it twice.